app/dashboards/accession_dashboard.rb

```diff
@@ -0,0 +1,66 @@
+require 'administrate/base_dashboard'
+
+class AccessionDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    degree_period: Field::BelongsTo.with_options(collection: DegreePeriod::VALID_GRAD_MONTHS),
+    accession_number: Field::String,
+    created_at: Field::DateTime,
+    updated_at: Field::DateTime
+  }.freeze
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = %i[
+    id
+    degree_period
+    accession_number
+    created_at
+  ].freeze
+
+  # SHOW_PAGE_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's show page.
+  SHOW_PAGE_ATTRIBUTES = %i[
+    id
+    degree_period
+    accession_number
+    created_at
+    updated_at
+  ].freeze
+
+  # FORM_ATTRIBUTES
```

Comment:
I think this would require custom `new` and `edit` views. Is that alright with you? I'm not opposed to it, but I want to make sure it's worth the additional debt.

Reply:
So I think for now let's not fork the templates and remind/inform stakeholders they need to make sure documentation exists and is accurate someplace. If they ask us to add it to those forms, we can fork though.